run_service.sh

```diff
@@ -351,6 +378,7 @@ try_read_storage() {
             fi
         done
 
+        source "$env_file_path"
         rpc=$(cat $rpc_path)
         agent_address=$(cat $agent_address_path)
         operator_address=$(get_address "$operator_keys_file")
```

Comment:
This approach is inconsistent.

Reply:
What do you suggest here?

Reply:
I mean rpc, agent address etc. are in text files and then are read in a variable. We should either also add these in the env file or use the same approach with the staking flag so that the code is consistent.

Let's not address this now though.